includes/fbbot/class-omise-fbbot-user-service.php

```diff
@@ -0,0 +1,86 @@
+<?php
+defined( 'ABSPATH' ) or die( "No direct script access allowed." );
+
+if ( class_exists( 'Omise_FBBot_User_Service' ) ) {
+  return;
+}
+
+class Omise_FBBot_User_Service {		
+	private function __construct() {
+		// Hide the constructor
+	}
+
+	public static function get_user( $sender_id ) {
+		$user = get_user_by( 'login', $sender_id );
+
+		if ($user) {
+			return $user->data;
+		}
+
+		$fb_profile = self::get_fb_profile( $sender_id );
+		if ( ! $fb_profile ) 
+			return NULL;
+
+		$wp_user = self::create_wp_user( $fb_profile );
```

Comment:
Good catch krub, I create a new user because just need to create a username by use MessengerID for use in the future if we have a new feature like publish message from WooCommerce admin backend to the customer on Messenger App.
I just prepare for it. But if it not useful in this version i can remove it. 🙇